src/rest_api/classes/generic.clj

```diff
@@ -0,0 +1,52 @@
+(ns rest-api.classes.generic
+  (:require
+    [pseudoace.utils :as pace-utils]
+    [rest-api.formatters.object :as obj :refer  [pack-obj]]
+    [clojure.string :as str]))
+
+(defn name-field [object]
+    (obj/name-field object))
+
+(defn xform-species-name
+    "Transforms a `species-name` from the WB database into
+      a name used to look up connection configuration to a sequence db."
+        [species]
+          (let  [species-name-parts (str/split species #" ")
+                          g  (str/lower-case  (ffirst species-name-parts))
+                                   species  (second species-name-parts)]
+                (str/join "_"  [g species])))
+
+(defn xrefs [object]
+  (let [data
+        (if-let [k (first (filter #(= (name %) "id") (keys object)))]
+          (let [role (namespace k)
+                ckw (partial (str/join "." [role "database"]))]
```

Comment:
This is what `eastwood` is hinting is wrong, it's an expression returns a function, where it's intended usage appears to be as a string (used as first argument to `keyword`), thus I believe should just be `ckw (str role ".database")`.